flytekit/core/tracker.py

```diff
@@ -337,6 +337,10 @@ def extract_task_module(f: Union[Callable, TrackedInstance]) -> Tuple[str, str,
 
 
 def get_full_module_path(mod: ModuleType, mod_name: str) -> str:
+    from flytekit.tools.interactive import ipython_check
+
+    if ipython_check():
+        return ""
```

Comment:
Do we really need it here? I feel like we can remove it. We already pickle the task entity, so `get_full_module_path` should not be called.